build/package/bitbucket-pipelines/bitbucket-pipelines.go

```diff
@@ -1,49 +1,34 @@
 package main
```

Comment:
All of the changes in `build/package` are changes to wrappers to be compatible with the migration to cobra. I'd recommend looking at these last